### ISSUE TITLE
How to interpret predicted KITTI label output

### ISSUE THREAD

**Issue by Hunkzer** [traveller59/second.pytorch#160]
Hello!

I implemented the [PointPillars](https://github.com/nutonomy/second.pytorch) solution and since it depends on the SECOND code base, I'm asking for some advice here:

I compared the labels from the evaluation against ground truth labels from the official KITTI data set:

![Screenshot from 2019-04-16 10-07-40](https://user-images.githubusercontent.com/49489817/56193436-eea50680-6030-11e9-90c7-a742b24d6064.png)

Why are there so much more cars in the predicted file? Is my implementation working as intended? I appreciate your help :)

~Hunkzer

**Comment by traveller59** [comment]
these cars are low-score cars and they won't harm performance. If your detector have too much false negatives, you can decrease the score threshold to decrease false negatives (increase false positives) and increase KITTI AP.
In addition, I have trained a xyres_16 model in this project but only get 77.4 val 3D AP (77.9 in their paper).

**Comment by Hunkzer** [comment]
Thank you for your very fast response!
